docuilib/src/pages/livePreview.module.css

```diff
@@ -0,0 +1,6 @@
+.mobileFlexContainer {
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+  padding: 20px;
```

Comment:
Don't we want the "user" to decide on the `padding`? Could we have a screen that wants no `padding`?